<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Madde::Internal::MaemoPackageCreationWidget</class>
 <widget class="QWidget" name="Madde::Internal::MaemoPackageCreationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="packageNameLabel">
       <property name="text">
        <string>Package name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="packageNameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Package version:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Major:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="major">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Minor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Patch:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="patch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="shortDescriptionLabel">
       <property name="text">
        <string>Short package description:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="shortDescriptionLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="packageManagerNameLabel">
       <property name="text">
        <string>Name to be displayed in Package Manager:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="packageManagerNameLineEdit"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="packageManagerIconLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Icon to be displayed in Package Manager:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QToolButton" name="packageManagerIconButton">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="packageManagerIconSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="editDebianFileLabel">
       <property name="text">
        <string>Adapt Debian file:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="debianFilesComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="editDebianFileButton">
         <property name="text">
          <string>Edit...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="editSpecFileButton">
       <property name="text">
        <string>Edit spec file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>major</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Madde::Internal::MaemoPackageCreationWidget</receiver>
   <slot>versionInfoChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Madde::Internal::MaemoPackageCreationWidget</receiver>
   <slot>versionInfoChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patch</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Madde::Internal::MaemoPackageCreationWidget</receiver>
   <slot>versionInfoChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDebianFileButton</sender>
   <signal>clicked()</signal>
   <receiver>Madde::Internal::MaemoPackageCreationWidget</receiver>
   <slot>editDebianFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>packageManagerIconButton</sender>
   <signal>clicked()</signal>
   <receiver>Madde::Internal::MaemoPackageCreationWidget</receiver>
   <slot>setPackageManagerIcon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addFile()</slot>
  <slot>removeFile()</slot>
  <slot>handleSkipButtonToggled(bool)</slot>
  <slot>versionInfoChanged()</slot>
  <slot>editDebianFile()</slot>
  <slot>setPackageManagerIcon()</slot>
 </slots>
</ui>
